department-of-veterans-affairs/vets-website
# Blue Button PDF cover page enhancements

## Are you removing, renaming or moving a folder in this PR?

### :warning: TeamSites :warning:
Did you change site-wide styles, platform utilities or other infrastructure?

## Summary

Updates the Blue Button PDF cover page to include the user's name, date range, and records available and unavailable. Additionally changes the download page to show an alert and prevent download when no records would be included in the download. (See [slack thread](https://dsva.slack.com/archives/C03Q2UQL1AS/p1734037299624789) for details)

Note: the data last updated 

## Related issue(s)

- https://jira.devops.va.gov/browse/MHV-60682
- https://dsva.slack.com/archives/C03Q2UQL1AS/p1734037299624789
- figma: https://www.figma.com/design/SGP1z2LejUWqDZyT61po5J/Medical-Records---Phase-1?node-id=171-9856&node-type=canvas&t=Dckn31WNgip77Jcb-0

## Testing Instructions

- All categories have records
  - select "last 12 months"
  - select "care summaries & notes" and "vaccines"
  - [ ] download button is shown
  - [ ] PDF has "Records in this report" section with "care summaries & notes", "vaccines" and "My HealtheVet account summary" listed
  - [ ] PDF does not have "Records not in this report" section
- Not all categories have records
  - select "last 12 months"
  - select "labs & test results" and "vaccines"
  - [ ] download button is shown
  - [ ] PDF has "Records in this report" section with "vaccines" and "My HealtheVet account summary" listed
  - [ ] PDF has "Records not in this report" section with "Lab and test results" listed
- No categories have records
  - select "last 12 months"
  - select "labs & test results"
  - [ ] download button not is shown
  - [ ] No records found alert is shown
  - [ ] "Go back to update report" link is shown and takes you back to the date range selection

## Screenshots

![image (1)](https://github.com/user-attachments/assets/cfff12cd-2abe-43bd-8eed-7b55820214e7)
![image](https://github.com/user-attachments/assets/a868e295-5ffd-4f09-b9da-507be87f1d14)

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
